cmd/containerd-shim-runhcs-v1/exec_wcow_podsandbox.go

```diff
@@ -132,7 +132,7 @@ func (wpse *wcowPodSandboxExec) Start(ctx context.Context) error {
 
 	// Publish the task start event. We mever have an exec for the WCOW
 	// PodSandbox.
-	wpse.events.publishEvent(
+	_ = wpse.events.publishEvent(
```

Comment:
I think we should just return this error. If we fail to publish an event upstream components could be affected.

Reply:
Perhaps fix the typo "mever" in the comment while we're here, too.